### DISCUSSION TITLE
[react-markdown] ReactMarkdown does not render or display blog content

### DISCUSSION THREAD

**Discussion by Trushar10** [remarkjs/remark#727] [Q&A] [answered]
in the blog-template.js, using only ```{ content }``` displays the blog content. But using the ```ReactMarkdown``` component displays nothing.

My repository link: [portfolio](https://github.com/Trushar10/portfolio)

![Screenshot (94)](https://user-images.githubusercontent.com/63665542/119709288-a9439c00-be7a-11eb-969d-e0e6e922141c.png)

using only ```{ content }``` displays

```
# JavaScript Hard Parts ## Subtitle - first item - second item - third item > I'm baby dIY beard kogi tumeric butcher freegan. Schlitz vaporware everyday carry small batch, taxidermy deep v glossier iceland truffaut retro asymmetrical bitters tattooed post-ironic vape. Jean shorts blog edison bulb, yr tousled banjo lumbersexual 3 wolf moon squid artisan poutine typewriter. Cray pinterest master cleanse helvetica, godard heirloom taiyaki kinfolk hell of locavore. ``` allStrapiProjects { nodes { github id description title ```
```

I had not modified any text. Only uncommented it.

blog-template.js
```
import React from "react"
import { graphql, Link } from "gatsby"
import Layout from "../components/Layout"
import ReactMarkdown from "react-markdown"

const ComponentName = ({ data }) => {
  const { content } = data.blog

  return (
    <Layout>
      <section className="blog-template">
        <div className="section-center">
          <article className="blog-content">
            <ReactMarkdown source={content}></ReactMarkdown>
          </article>
          <Link to="/blog" className="btn center-btn">
            blog
          </Link>
        </div>
      </section>
    </Layout>
  )
}

export const query = graphql`
  query GetSingleBlog($slug: String) {
    blog: strapiBlogs(slug: { eq: $slug }) {
      content
    }
  }
`

export default ComponentName
```

**Comment by ChristianMurphy** [comment]
@Trushar10 check what the value of `content` is with
```ts
console.log(content)
```
it's likely Gatsby is modifying/escaping content before it gets to `react-markdown`

**Reply by Trushar10** [reply]
I am getting the content with ```console.log(content)```.

![Screenshot (96)](https://user-images.githubusercontent.com/63665542/120077853-0ed99780-c0ca-11eb-806c-20f51f0b1d76.png)

**Comment by wooorm** [comment, marked as answer]
https://github.com/remarkjs/react-markdown/blob/main/changelog.md#change-source-to-children

**Reply by Trushar10** [reply]
Thank you. Why it is changed from source to children?

**Reply by wooorm** [reply]
You should be able to find more background by searching the issues. In short: children has meaning in react
